components/editor/netbeans/Makefile

```diff
@@ -52,8 +57,15 @@ $(BUILD_DIR_64)/.installed: $(BUILD_DIR_64)/.built
 	touch $(BUILD_DIR_64)/.installed;
 
 install: $(BUILD_DIR_64)/.installed
+	$(GSED) -i -e  "1s?.*?$(PYTHON_SHEBANG)?" $(PROTO_DIR)/usr/netbeans/$(COMPONENT_VERSION)/extide/ant/bin/runant.py;
 
+# Runtime dependencies
```

Comment:
Runtime-only dependencies are not needed to be manually listed in `REQUIRED_PACKAGES`.